src/vs/workbench/browser/parts/activitybar/activitybarPart.ts

```diff
@@ -187,6 +187,13 @@ export class ActivitybarPart extends Part implements IActivityBarService {
 					async () => { this.instantiationService.invokeFunction(accessor => new ToggleActivityBarVisibilityAction().run(accessor)); }
 				));
 
+				const toggleSidebarPositionAction = new ToggleSidebarPositionAction(
```

Comment:
you can shorten this by leveraging the instantiation service as seen on line 168. also, I'm curious if you prefer the action above or below the `Hide Activity Bar` action?

Reply:
It'll make more sense to have the move action above the `Hide Activity Bar` action, because in other cases, the movement actions come before the hide action. Will refactor and update this commit.